raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -256,7 +256,7 @@ contract MonitoringService is Utils {
 
         // We don't actually know when the channel has been closed. So we'll
         // make a guess so that assumed_close_block >= real_close_block.
-        uint256 assumed_settle_timeout = token_network.settlement_timeout_min();
+        uint256 assumed_settle_timeout = token_network.settle_timeout();
```

Comment:
Nice, this even increases accuracy! We do know the exact settle time now, don't we? If that's the case, the comment and variable name should be updated to reflect that.

Reply:
Good point, this went unnoticed ✅